### ISSUE TITLE
tracker: Fedora 37 changes considerations

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-tracker#1222]
*Output generated by and [stored alongside](https://pagure.io/fork/dustymabe/fedora-pgm/pgm_scripts/blob/dusty-fcos-changes/f/changes/FCOS-changes.md.orig) ([with modifications](https://pagure.io/fork/dustymabe/fedora-pgm/pgm_scripts/blob/dusty-fcos-changes/f/changes/FCOS-changes.md)) [this](https://pagure.io/fork/dustymabe/fedora-pgm/pgm_scripts/blob/dusty-fcos-changes/f/changes/genFCOSChanges.sh) script in a fork of the [pgm_scripts repo](https://pagure.io/fedora-pgm/pgm_scripts).*

---

**Fedora 37 Accepted System-Wide Changes** ([wiki source](https://fedoraproject.org/wiki/Releases/37/ChangeSet#Fedora_Linux_37_Accepted_System-Wide_Changes))

101. ~~✔️ [DNS Over TLS](https://fedoraproject.org/wiki/Changes/DNS_Over_TLS)~~
     - ~~Fedora will attempt to use DNS over TLS (DoT) if supported by configured DNS servers.~~
     - ~~Tracking bug: [#1889901](https://bugzilla.redhat.com/show_bug.cgi?id=1889901)~~
     - ~~**NOTES**: **JL**: no special considerations necessary for FCOS, but be on the lookout for DNS-related fallout~~
        - ~~F35 changes issue: https://github.com/coreos/fedora-coreos-tracker/issues/873~~
101. ~~✔️ [DNF/RPM Copy on Write enablement for all variants](https://fedoraproject.org/wiki/Changes/RPMCoW)~~
     - ~~RPM Copy on Write provides a better experience for Fedora Users as it reduces the amount of I/O and offsets CPU cost of package decompression. RPM Copy on Write uses reflinking capabilities in btrfs, which is the default filesystem starting from Fedora 33 for most variants. Note that this behavior is not being turned on by default for this Change.~~
     - ~~Tracking bug: [#1915976](https://bugzilla.redhat.com/show_bug.cgi?id=1915976)~~
     - ~~**NOTES**: **JL**: this path of librpm is not used by rpm-ostree. the whole download and unpack path is ostree native and has different tradeoffs. good to keep track of this conceptually, but nothing for FCOS to do here.~~
101. ✔️ [Python 3.11](https://fedoraproject.org/wiki/Changes/Python3.11)
     - Update the Python stack in Fedora from Python 3.10 to Python 3.11, the newest major release of the Python programming language.
     - Tracking bug: [#2016048](https://bugzilla.redhat.com/show_bug.cgi?id=2016048)
     - **NOTES**: **JL**: we don't ship Python. but might be worth going over Python 3.11 release notes for Python-based codebases like cosa for e.g. deprecations.
101. ✔️ ~~[Drop NIS(+) support from PAM](https://fedoraproject.org/wiki/Changes/drop_NIS_support_from_PAM)~~
     - ~~This change is about dropping user-authentication using NIS(+) from PAM.~~
     - ~~Tracking bug: [#2021660](https://bugzilla.redhat.com/show_bug.cgi?id=2021660)~~
     - ~~**NOTES**: **JL**: may affect users who use NIS+? likely not though. If so, we should direct them to e.g. LDAP or FreeIPA as the Change proposal suggests. so overall, skip.~~
101. ✔️~~ [Retire the NIS(+) user-space utility programs](https://fedoraproject.org/wiki/Changes/retire_NIS_user_space_utils)~~
     - ~~This change is about retiring the ypbind, yp-tools, and ypserv packages, and removal of the {nis,yp}domainname user-space utility programs from the hostname package.~~
     - ~~Tracking bug: [#2022386](https://bugzilla.redhat.com/show_bug.cgi?id=2022386)~~
     - ~~**NOTES**: **JL**: may affect users who use NIS+? likely not though. If so, we should direct them to e.g. LDAP or FreeIPA as the Change proposal suggests. so overall, skip.~~
101. ✔️ [ELN-Extras](https://fedoraproject.org/wiki/Changes/ELN-Extras)
     - ELN-extras will be a new build target and compose similar in behavior to ELN, but closer to EPEL in function. It will be a place to prepare and maintain packages that may be desired for EPEL N+1 while RHEL N+1 is still being incubated in ELN.
     - Tracking bug: [#2028161](https://bugzilla.redhat.com/show_bug.cgi?id=2028161)
     - **NOTES**: **JL**: not directly relevant to FCOS.
101. ✔️ [RetireARMv7](https://fedoraproject.org/wiki/Changes/RetireARMv7)
     - Retire the ARMv7 architecture AKA arm32 or armhfp architecture.
     - Tracking bug: [#2028172](https://bugzilla.redhat.com/show_bug.cgi?id=2028172)
     - Notes: **DM**: FCOS doesn't ship ARMv7. Nothing to do.
101. ✔️ [MinGW toolchain update](https://fedoraproject.org/wiki/Changes/F37MingwEnvToolchainUpdate)
     - Update the MinGW toolchain to the latest upstream stable releases.
     - Tracking bug: [#2060050](https://bugzilla.redhat.com/show_bug.cgi?id=2060050)
     - **NOTES**: **DM**: skipping MinGW environment and toolchain update because it's a Windows cross-compiler, not relevant.
101. ✔️ [Boost 1.78 upgrade](https://fedoraproject.org/wiki/Changes/F37Boost178)
     - This change brings Boost 1.78 to Fedora. This will mean Fedora ships with a recent upstream Boost release.
     - Tracking bug: [#2062915](https://bugzilla.redhat.com/show_bug.cgi?id=2062915)
     - **NOTES**: **DM**: skipping Boost 1.78 upgrade because it should be contained to the build system (making sure dependent packages compile)
101. ⚠️  [Signed RPM Contents](https://fedoraproject.org/wiki/Changes/Signed_RPM_Contents)
     - We want to add signatures to individual files that are part of shipped RPMs.
     - Tracking bug: [#2076296](https://bugzilla.redhat.com/show_bug.cgi?id=2076296)
     - **NOTES**: **JL**: Nothing required of us for this but we may want to take advantage. IMA-related features in the future. Colin already started doing some work for this in rpm-ostree (https://github.com/coreos/rpm-ostree/pull/3610). See also recent ostree-composefs work: https://blogs.gnome.org/alexl/2022/06/02/using-composefs-in-ostree/
101. ✔️ ~~[Changes/LegacyXorgDriverRemoval](https://fedoraproject.org/wiki/Changes/LegacyXorgDriverRemoval)~~
     - ~~This change removes the xorg-x11-drv-vesa and xorg-x11-drv-fbdev driver packages, and associated support code from the xorg-x11-server-Xorg package.~~
     - ~~Tracking bug: [#2078921](https://bugzilla.redhat.com/show_bug.cgi?id=2078921)~~
     - ~~**NOTES**: **DM**: FCOS doesn't ship a graphical desktop. This shouldn't affect us.~~
101. ️✔️  [RPM 4.18](https://fedoraproject.org/wiki/Changes/RPM-4.18)
     - Update RPM to the 4.18 release.
     - Tracking bug: [#2079029](https://bugzilla.redhat.com/show_bug.cgi?id=2079029)
     - **NOTES**: **DM**: Nothing in particular jumps out that we need to update. 4.18 alpha has been in rawhide since april and our rawhide stream is building fine.
101. ✔️  [Drop i686 builds of jdk8,11,17 and latest (18) rpms from f37 onwards](https://fedoraproject.org/wiki/Changes/Drop_i686_JDKs)
     - java-1.8.0-openjdk, java-11-openjdk, java-17-openjdk and java-latest-openjdk packages will no longer build i686 subpackages
     - Tracking bug: [#2083750](https://bugzilla.redhat.com/show_bug.cgi?id=2083750)
     - **NOTES**: **LB**: we don't ship i686 images/packages in FCOS
101. ✔️  [Node.js 18.x by default](https://fedoraproject.org/wiki/Changes/Nodejs18)
     - The latest release of Node.js to carry a 30-month lifecycle is the 18.x series. As with 16.x, 14.x, 12.x, 10.x and 8.x before it, Fedora 37 will carry 18.x as the default Node.js interpreter for the system. The 16.x, and 14.x interpreters will remain available as non-default module streams.
     - Tracking bug: [#2087185](https://bugzilla.redhat.com/show_bug.cgi?id=2087185)
     - **NOTES**: **LB**: we don't ship NodeJS
101. ✔️  [Perl 5.36](https://fedoraproject.org/wiki/Changes/perl5.36)
     - A new perl 5.36 version brings a lot of changes done over a year of development. Perl 5.36 will be released in May 28th 2022. See perldelta for more details about new release.
     - Tracking bug: [#2088002](https://bugzilla.redhat.com/show_bug.cgi?id=2088002)
     - **NOTES**: **LB**: we don't ship Perl
101. ⚠️  [Strong crypto settings: phase 3, forewarning 1/2](https://fedoraproject.org/wiki/Changes/StrongCryptoSettings3Forewarning1) 👉 https://github.com/coreos/fedora-coreos-tracker/issues/1230
     - Cryptographic policies will be tightened in Fedora 38-39, SHA-1 signatures will no longer be trusted by default.  Fedora 37 specifically doesn't come with any change of defaults, and this Fedora Change is an advance warning filed for extra visibility.  Test your setup with TEST-FEDORA39 today and file bugs so you won't get bit by Fedora 38-39.
     - Tracking bug: [#2089811](https://bugzilla.redhat.com/show_bug.cgi?id=2089811)
     - **NOTES**: **LB**: we may want to check how much we will be impact by this in F38/F39
101. ✔️  [Python: Add -P to default shebangs](https://fedoraproject.org/wiki/Changes/PythonSafePath)
     - The -P flag will be added to the Python shebang macros (%{py3_shbang_opts}, %{py3_shebang_flags}, ...). Packages that adhere to those macros will change their Python shebangs from #! /usr/bin/python3 -s to #! /usr/bin/python3 -sP and as a result, will no longer have the directory of the script (such as /usr/bin) in sys.path. An opt-out mechanism exists.
     - Tracking bug: [#2090866](https://bugzilla.redhat.com/show_bug.cgi?id=2090866)
     - **NOTES**: **LB**: we don't ship Python packages
101. ⚠️  [BIOS boot.iso with GRUB2](https://fedoraproject.org/wiki/Changes/BIOSBootISOWithGrub2) 👉 https://github.com/coreos/fedora-coreos-tracker/issues/1231
     - Modify lorax-generic-templates to use GRUB2 when booting the boot.iso on BIOS systems, instead of syslinux. Upstream syslinux development is dead, and the Fedora maintainer would like to drop the package from the distribution. GRUB2 works as a replacement in most situations and continues to have upstream support.
     - Tracking bug: [#2092065](https://bugzilla.redhat.com/show_bug.cgi?id=2092065)
     - **NOTES**: **LB**: I think we'd like to do this move too, in coreos-assembler and FCOS-config. Also consider what happens for RHCOS.
101. ✔️  [Install Using GPT on x86_64 BIOS by Default](https://fedoraproject.org/wiki/Changes/GPTforBIOSbyDefault)
     - This Change makes it so that Fedora Linux systems installed on legacy x86 BIOS systems will get GPT partitioning by default instead of legacy MBR partitioning. This makes x86 BIOS installs more similar to x86 UEFI installs.
     - Tracking bug: [#2092091](https://bugzilla.redhat.com/show_bug.cgi?id=2092091)
     - **NOTES**: **LB**: We do already have this kind of disk layout in FCOS. 
101. ✔️  [Return Cloud Base to Edition Status](https://fedoraproject.org/wiki/Changes/RestoreCloudEdition)
     - Cloud should be listed on getfedora.org with Workstation, Server and IoT. The petition to reinstate the Cloud Base as an official Edition is based on the clear identification of unique environmental support requirements for private hyperscaler and public cloud environments not specifically addressed by other Editions.
     - Tracking bug: [#2096419](https://bugzilla.redhat.com/show_bug.cgi?id=2096419)
     - **NOTES**: **DM**: This only affects the Fedora Cloud edition.
101. ✔️ ~~[Build all JDKs in Fedora against in-tree libraries and with static stdc++lib](https://fedoraproject.org/wiki/Changes/JdkInTreeLibsAndStdclibStatic)~~
     - ~~This is initial step to move JDKs to be more like other JDKs, to build proper transferable images, and to lower certification burden of each binary. Long storyshort, first step in: https://fedoraproject.org/wiki/MoveFedoraJDKsToBecomePortableJDKs~~
     - ~~Tracking bug: [#2097516](https://bugzilla.redhat.com/show_bug.cgi?id=2097516)~~
     - ~~**NOTES**: **DM**: We don't ship java~~
101. ~~✔️  [RPM Macros for Build Flags](https://fedoraproject.org/wiki/Changes/RPMMacrosForBuildFlags)~~
     - ~~Create a corresponding macro for each compiler flag in the redhat-rpm-config macro file and create "extra flag" macros to make it easier for packages to add and remove compiler flags.~~
     - ~~Tracking bug: [#2100610](https://bugzilla.redhat.com/show_bug.cgi?id=2100610)~~
     - ~~**NOTES**: **DM**: We may want to update our packages, but no need to update anything now.~~
101. ✔️  [Gettext Runtime Subpackage](https://fedoraproject.org/wiki/Changes/GettextRuntimeSubpackage)
     - Subpackage gettext's runtime programs in a small runtime subpackage, reducing the default install footprint.
     - Tracking bug: [#2103239](https://bugzilla.redhat.com/show_bug.cgi?id=2103239)
     - **NOTES**: **DM**: Nothing to do. If the packages in FCOS are updated to just require the gettext-runtime package then we'll get the space savings as well.
101. ✔️  [Golang 1.19](https://fedoraproject.org/wiki/Changes/golang1.19)
     - Rebase of Golang package to upcoming version 1.19 in Fedora 37, including the rebuild of all dependent packages(the pre-release version of Go will be used for the rebuild if released version will not be available at the time of the mass rebuild).
     - Tracking bug: [#2103240](https://bugzilla.redhat.com/show_bug.cgi?id=2103240)
     - **NOTES**: **DM**: We may want to update the CI for our repos, but nothing to do at the FCOS level. Rawhide will pick it up first and notify us of problems.
101. ✔️  [Fallback Hostname](https://fedoraproject.org/wiki/Changes/FallbackHostname)
     - This proposal is for the fallback hostname for server like variants of Fedora to use localhost as the fallback hostname.
     - Tracking bug: [#2104276](https://bugzilla.redhat.com/show_bug.cgi?id=2104276)
     - **NOTES**: **DM**: This is our change proposal. See https://github.com/coreos/fedora-coreos-tracker/issues/902
101. ✔️  [libsoup 3: Part One](https://fedoraproject.org/wiki/Changes/libsoup_3:_Part_One)
     - libsoup 3 is a new API version of libsoup that provides support for HTTP/2. Unfortunately, it is incompatible with libsoup 2. To avoid misbehavior, applications will crash on startup if linked to both libsoup 2 and libsoup 3 at the same time. Because many libraries depend on libsoup, and applications have limited control over which libsoup they link to transitively, this transition will be tricky and requires attention from all Fedora packages that depend on libsoup, even if only indirectly.
     - Tracking bug: [#2105119](https://bugzilla.redhat.com/show_bug.cgi?id=2105119)
     - **NOTES**: **JL**: Nothing to do. We don't have any packages in FCOS that require or pull in libsoup.
101. ✔️  [Make Fedora CoreOS a Fedora Edition](https://fedoraproject.org/wiki/Changes/FedoraCoreOS)
     - This change is to promote Fedora CoreOS to Edition status alongside Cloud, IoT, Server and Workstation.
     - Tracking bug: [#2106823](https://bugzilla.redhat.com/show_bug.cgi?id=2106823)
     - **NOTES**: **DM**: This is our change proposal. See https://github.com/coreos/fedora-coreos-tracker/issues/915
101. ✔️  [IBus 1.5.27](https://fedoraproject.org/wiki/Changes/IBus_1.5.27)
     - In IBus 1.5.27, ibus restart subcommand will be enhanced to be able to restart ibus-daemon in GNOME desktop, ibus im-module subcommand will be added to get internal gtk-im-module value in an GTK instance, ibus-setup will provides custom themes for the IBus candidate window.
     - Tracking bug: [#2107753](https://bugzilla.redhat.com/show_bug.cgi?id=2107753)
     - **NOTES**: **JL**: Nothing to do. We don't ship IBus.
101. ~~✔️  [MAC Address Policy none](https://fedoraproject.org/wiki/Changes/MAC_Address_Policy_none)~~
     - ~~The systemd-udev package installs "/usr/lib/systemd/network/99-default.link", which sets Link.MACAddressPolicy=persistent for all software NIC devices. This proposal is to add to the policy so that we use Link.MACAddressPolicy=none for bond/bridge/team devices.~~
     - ~~Tracking bug: [#2107754](https://bugzilla.redhat.com/show_bug.cgi?id=2107754)~~
     - ~~**NOTES**: **DM**: This is our change proposal. See https://github.com/coreos/fedora-coreos-tracker/issues/919~~
101. ✔️  [Firefox Langpacks Subpackage](https://fedoraproject.org/wiki/Changes/Firefox_Langpacks_Subpackage)
     - Firefox langpacks, which have been bundled in the Fedora firefox base package until now, will be moved to a firefox-langpacks subpackage.
     - Tracking bug: [#2108184](https://bugzilla.redhat.com/show_bug.cgi?id=2108184)
     - **NOTES**: **DM**: Nothing to do. We don't ship Firefox.
101. ✔️  [GNU Toolchain Update (glibc 2.36, binutils 2.38)](https://fedoraproject.org/wiki/Changes/GNUToolchainF37)
     - Update the Fedora 37 GNU Toolchain to glibc 2.36, binutils 2.38.
     - Tracking bug: [#2108675](https://bugzilla.redhat.com/show_bug.cgi?id=2108675)
     - **NOTES**: **JL**: Should be nothing for us to do here.
101. ✔️  [Deprecate openssl1.1 package](https://fedoraproject.org/wiki/Changes/DeprecateOpensslCompat)
     - We are going to deprecate openssl1.1 package following the guidelines for deprecated packages: https://docs.fedoraproject.org/en-US/packaging-guidelines/deprecating-packages/.
     - Tracking bug: [#2108694](https://bugzilla.redhat.com/show_bug.cgi?id=2108694)
     - **NOTES**: **JL**: Nothing for us to do. We don't ship openssl1.1.

---

**Fedora 37 Accepted Self-Contained Changes** ([wiki source](https://fedoraproject.org/wiki/Releases/37/ChangeSet#Fedora_Linux_37_Accepted_Self-Contained_Changes))

201. ~~✔️  [glibc 32 Build Adjustments](https://fedoraproject.org/wiki/Changes/glibc32_Build_Adjustments)~~
     - ~~The glibc32 package is a special package used by gcc and a few other packages to work around the lack of RPM multilib repository support in Koji. It is difficult to maintain, and the current approach raises questions regarding (L)GPL compliance.~~
     - ~~Tracking bug: [#1598524](https://bugzilla.redhat.com/show_bug.cgi?id=1598524)~~
     - ~~**NOTES**: **JL**: none of our components use glibc32 AFAIK, so should be good to skip.~~
201. ~~✔️  [Modular GNOME Keyring services](https://fedoraproject.org/wiki/Changes/ModularGnomeKeyring)~~
     - ~~The monolithic daemon provided by GNOME Keyring will be split into dedicated sub-daemons, so that they can be consistently managed by systemd.~~
     - ~~Tracking bug: [#1899998](https://bugzilla.redhat.com/show_bug.cgi?id=1899998)~~
     - ~~**NOTES**: *LB**: we don't ship these GNOME services in FCOS~~
201. ~~✔️  [Wayland by Default for SDDM](https://fedoraproject.org/wiki/Changes/WaylandByDefaultForSDDM)~~
     - ~~Change the default display server mode for SDDM to use a Wayland-based greeter rather than an X11-based one.~~
     - ~~Tracking bug: [#2050378](https://bugzilla.redhat.com/show_bug.cgi?id=2050378)~~
     - ~~**NOTES**: **LB**: we don't ship `sddm` in FCOS~~
201. ✔️  [Python Dist RPM provides to only provide PEP503-normalized names](https://fedoraproject.org/wiki/Changes/PythonDistPEP503ProvidesOnly)
     - The legacy python3dist(NAME) and python3.11dist(NAME) RPM provides with dots (.) in NAME will no longer be automatically provided. NAME will only be normalized according to PEP 503. E.g. on Fedora 36 a package provides both python3dist(ruamel-yaml) and python3dist(ruamel.yaml), on Fedora 37+ it will only provide python3dist(ruamel-yaml) (and similarly, python3.11dist(ruamel-yaml)).
     - Tracking bug: [#2052217](https://bugzilla.redhat.com/show_bug.cgi?id=2052217)
     - **NOTES**: **LB**: we don't ship Python in FCOS
201. ✔️  [MinGW UCRT target](https://fedoraproject.org/wiki/Changes/F37MingwUCRT)
     - This proposal is to add the UCRT target & support from Fedora to the MinGW cross-toolchains.
     - Tracking bug: [#2055254](https://bugzilla.redhat.com/show_bug.cgi?id=2055254)
     - **NOTES**: **LB**: mingw is not a concern for FCOS
201. ✔️  [MinGW OpenSSL 3.x update](https://fedoraproject.org/wiki/Changes/F37MingwOpenSSL3)
     - Update OpenSSL for MinGW to version 3.x
     - Tracking bug: [#2055724](https://bugzilla.redhat.com/show_bug.cgi?id=2055724)
     - **NOTES**: **LB**: mingw is not a concern for FCOS
201. ⚠️  [Enable read only /sysroot for Fedora Silverblue & Kinoite](https://fedoraproject.org/wiki/Changes/Silverblue_Kinoite_readonly_sysroot) 👉 https://github.com/coreos/fedora-coreos-tracker/issues/1232
     - This change is about enabling an opt-in ostree feature that re-mounts /sysroot as read only to avoid accidental changes.
     - Tracking bug: [#2060976](https://bugzilla.redhat.com/show_bug.cgi?id=2060976)
     - **NOTES**: **LB**: we did this in FCOS early on, but we never migrated older instances
201. ✔️  [Encourage Dropping Unused / Leaf Packages on i686](https://fedoraproject.org/wiki/Changes/EncourageI686LeafRemoval)
     - Package maintainers are empowered to stop building their packages for i686 — especially if supporting this architecture requires significant investment of time or resources. This will not apply to packages which are still depended on by other i686 packages, or which get used in a "multilib" context (i.e. for running 32-bit applications on x86_64). Dropping i686 architecture support from a leaf package will no longer be considered a breaking change, will not require any announcements, or tracker bugs.
     - Tracking bug: [#2069738](https://bugzilla.redhat.com/show_bug.cgi?id=2069738)
     - **NOTES**: **JL**: we don't explicitly try to support i686, though we do build it by default for various components. we could look at explicitly disabling it, or just ride the wave until Fedora itself turns it off.
201. ⚠️  [Support FIDO Device Onboarding](https://fedoraproject.org/wiki/Changes/FIDODeviceOnboarding)
     - Package and enable the FIDO Device Onboarding software stack for Zero Touch Onboarding on Fedora IoT.
     - Tracking bug: [#2075529](https://bugzilla.redhat.com/show_bug.cgi?id=2075529)
     - **NOTES**: **LB**: not exactly the same, but we may look into this and possibly file a feature-request for ourselves for Ignition
201. ✔️  [Build Fedora IoT Artifacts with osbuild](https://fedoraproject.org/wiki/Changes/IoTArtifactsWithOSBuild)
     - Build the key Fedora IoT artifacts such as the raw images and the traditional anaconda installer with osbuild.
     - Tracking bug: [#2075530](https://bugzilla.redhat.com/show_bug.cgi?id=2075530)
     - **NOTES**: **LB**: IoT specific
201. ~~[Haskell GHC 9.0 & Stackage LTS 19](https://fedoraproject.org/wiki/Changes/Haskell_GHC_9.0_&_Stackage_19)~~
     - ~~The GHC Haskell compiler will be updated from major version 8.10 to 9.0, and Haskell packages will be updated from Stackage LTS 18 to LTS 19 versions.~~
     - ~~Tracking bug: [#2080355](https://bugzilla.redhat.com/show_bug.cgi?id=2080355)~~
     - ~~**NOTES**: **LB**: we don't ship GHC in FCOS~~
201. ✔️  [Replace jwhois package with whois for Fedora Workstation](https://fedoraproject.org/wiki/Changes/Replace_jwhois_with_whois_in_Fedora_Workstation)
     - Fedora Workstation product core group includes jwhois package. Replace it with whois package which is more actively developed.
     - Tracking bug: [#2081764](https://bugzilla.redhat.com/show_bug.cgi?id=2081764)
     - **NOTES**: **SA**: we don't ship jwhois
201. ✔️  [Enhance Persian Font Support](https://fedoraproject.org/wiki/Changes/EnhancePersianFontSupport)
     - This change aims to provide a consistent experience for those who use Fedora in Persian or write or read Persian text in Fedora.
     - Tracking bug: [#2095025](https://bugzilla.redhat.com/show_bug.cgi?id=2095025)
     - **NOTES**: **DM**: I don't think any changes for FCOS are needed here.
201. ✔️  [LLVM 15](https://fedoraproject.org/wiki/Changes/LLVM-15)
     - Update all llvm sub-projects in Fedora Linux to version 15.
     - Tracking bug: [#2100620](https://bugzilla.redhat.com/show_bug.cgi?id=2100620)
     - **NOTES**: **DM**: This shouldn't affect FCOS.
201. ✔️  [Supplement of Server distributables by a KVM VM disk image](https://fedoraproject.org/wiki/Changes/Supplement-server-by-kvm-vm-image)
     - Virtualization has long been a steadily growing use case of Fedora Server Edition, but it is still time consuming and tedious for the system administrator to create a Fedora Server VM. Supplementing the downloads by a KVM VM image remedies the deficiency.
     - Tracking bug: [#2100621](https://bugzilla.redhat.com/show_bug.cgi?id=2100621)
     - **NOTES**: **DM**: This is limited to the Server edition deliverables.
201. ✔️  [Erlang 25](https://fedoraproject.org/wiki/Changes/Erlang_25)
     - Update Erlang/OTP to version 25.
     - Tracking bug: [#2100623](https://bugzilla.redhat.com/show_bug.cgi?id=2100623)
     - **NOTES**: **DM**: This shouldn't affect FCOS.
201. ✔️  [Stratis 3.2.0](https://fedoraproject.org/wiki/Changes/Stratis_3.2.0)
     - Stratis 3.2.0 includes one significant enhancement, one bug fix, and a number of more minor improvements.
     - Tracking bug: [#2108166](https://bugzilla.redhat.com/show_bug.cgi?id=2108166)
     - **NOTES**: **JL**: Nothing do do. We don't ship Stratis.
201. ✔️  [LXQt 1.1.0](https://fedoraproject.org/wiki/Changes/LXQt_1.1)
     - Update LXQt to 1.1.0 in Fedora.
     - Tracking bug: [#2109614](https://bugzilla.redhat.com/show_bug.cgi?id=2109614)
     - **NOTES**: **DM**: Not applicable. We don't ship it.
201. ✔️  [Officially Support Raspberry Pi 4](https://fedoraproject.org/wiki/Changes/RaspberryPi4)
     - The work around Raspberry Pi 4 has been on going for a number of years, but we've never officially supported it due to lack of accelerated graphics and other key features. With Fedora 37, Raspberry Pi 4 is now officially supported, including accelerated graphics using the V3D GPU.
     - Tracking bug: [#2109648](https://bugzilla.redhat.com/show_bug.cgi?id=2109648)
     - **NOTES**: **DM**: No action for us to take. Hardware support is getting better in the kernel.
201. ⚠️  [Preset All Systemd Units on First Boot](https://fedoraproject.org/wiki/Changes/Preset_All_Systemd_Units_on_First_Boot) 👉 https://github.com/coreos/ignition/issues/1440
     - Systemd will execute the equivalent of systemctl preset-all when an unconfigured system is booted ("First Boot" condition). This means that units will be enabled or disabled according to the preset configuration. We currently do the equivalent of systemctl preset-all --preset-mode=enable-only, and this will be extended to also disable units, i.e. systemctl preset-all --preset-mode=full. Any units which are manually symlinked but presets say they shouldn't (which is against the packaging guidelines for packaged units) will be disabled.
     - Tracking bug: [#2114065](https://bugzilla.redhat.com/show_bug.cgi?id=2114065)
     - **NOTES**: **DM**: This will require some work for us. @jlebon filed https://github.com/coreos/ignition/issues/1440
201. ✔️  [Public release of the Anaconda Web UI preview image](https://fedoraproject.org/wiki/Changes/Anaconda_Web_UI_preview_image)
     - The work on Web UI for the Anaconda installer has advanced enough so that it is possible to create and publish self contained preview images.
     - Tracking bug: [#2114325](https://bugzilla.redhat.com/show_bug.cgi?id=2114325)
     - **NOTES**: **DM**: Not applicable. We don't ship anaconda.
201. ✔️  [BIND 9.18](https://fedoraproject.org/wiki/Changes/BIND_9.18)
     - Owner
     - Tracking bug: [#2114330](https://bugzilla.redhat.com/show_bug.cgi?id=2114330)
     - **NOTES**: **DM**: Not applicable. We don't ship bind (the server). We do ship bind-utils, but that should be fine.
201. ✔️  [ibus-libpinyin 1.13](https://fedoraproject.org/wiki/Changes/ibus-libpinyin_1.13)
     - This ibus-libpinyin release includes new features for English input and Table input.
     - Tracking bug: [#2114336](https://bugzilla.redhat.com/show_bug.cgi?id=2114336)
     - **NOTES**: **DM**: Not applicable. We don't ship ibus.
201. ✔️  [SELinux Parallel Autorelabel](https://fedoraproject.org/wiki/Changes/SELinux_Parallel_Autorelabel)
     - After a system's SELinux mode is switched from disabled to enabled, or after an administrator runs fixfiles onboot, SELinux autorelabel will be run in parallel by default.
     - Tracking bug: [#2114341](https://bugzilla.redhat.com/show_bug.cgi?id=2114341)
     - **NOTES**: **DM**: No action. We don't support auto-relabel.
201. ✔️  [Haskell GHC 8.10.7 & Stackage LTS 18.28](https://fedoraproject.org/wiki/Changes/Haskell_GHC_8.10.7)
     - The GHC Haskell compiler will be updated from minor version 8.10.5 to 8.10.7, and Haskell packages will be updated to Stackage LTS 18.28 minor versions.
     - Tracking bug: [#2115399](https://bugzilla.redhat.com/show_bug.cgi?id=2115399)
     - **NOTES**: **DM**: No action. We don't ship any related software.
201. ✔️  [Mumble 1.4](https://fedoraproject.org/wiki/Changes/Mumble1.4)
     - Update the Mumble voice chat application from 1.3 to 1.4.
     - Tracking bug: [#2115400](https://bugzilla.redhat.com/show_bug.cgi?id=2115400)
     - **NOTES**: **DM**: No action. We don't ship Mumble.
201. ✔️  [Emacs 28](https://fedoraproject.org/wiki/Changes/Emacs_28)
     - Update GNU Emacs to 28.1 release. This release includes a wide variety of new features, including native compilation of Lisp files.
     - Tracking bug: [#2118435](https://bugzilla.redhat.com/show_bug.cgi?id=2118435)
     - **NOTES**: **DM**: No action. We don't ship Emacs.
201. ✔️  [Minizip Renaming](https://fedoraproject.org/wiki/Changes/MinizipRenaming)
     - Renaming the "minizip" package to "minizip-ng" to align with the upstream naming.
     - Tracking bug: [#2129079](https://bugzilla.redhat.com/show_bug.cgi?id=2129079)
     - **NOTES**: **DM**: We don't ship minizip in FCOS, so this should be a NOOP for us.

**Comment by jlebon** [comment]
> [Enable read only /sysroot for Fedora Silverblue & Kinoite](https://fedoraproject.org/wiki/Changes/Silverblue_Kinoite_readonly_sysroot)
> - This change is about enabling an opt-in ostree feature that re-mounts /sysroot as read only to avoid accidental changes.
> - Tracking bug: [#2060976](https://bugzilla.redhat.com/show_bug.cgi?id=2060976)

A similar issue exists in FCOS BTW: the first official FCOS `stable` release was [January 11, 2020](https://github.com/coreos/fedora-coreos-streams/pull/45). It did not ship with read-only sysroot. That came in [May/June 2020](https://github.com/coreos/fedora-coreos-config/pull/304). So we have potentially really old FCOS `stable` nodes provisioned in those first 5-6 months (and of course `testing` too, which started even earlier) which today still don't have read-only sysroot.

It'd be nice to have some stats to know how many such nodes exist, but it likely isn't worth the effort/risk to try migrating them. Maybe instead we can add a CLHM dropin which suggests reprovisioning if read-only sysroot isn't active?

**Comment by dustymabe** [comment]
Updated the description with the outcomes from our small groups session earlier today.

**Comment by dustymabe** [comment]
We discussed the accepted changes today with a small group and then later in [the FCOS community meeting](https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/message/SGMUGRGYQ2E2IZSEMHYEQHKHRUAQF23R/). We found a few issues we wanted to discuss further and I have opened dedicated tickets for those discussions/investigations.

**Comment by dustymabe** [comment]
Reran the script and updated the description with new changes that have come in.

**Comment by dustymabe** [comment]
We discussed this in the [community meeting today](https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/message/LX2BMB6MWUS7TXHU5UZ4AHLOR2HGDKTG/).

We went over the newly added items in the changeset (121->125 and 214->216). I have updated the description with the results.

**Comment by dustymabe** [comment]
Reran the script and updated the description with new changes that have come in.

**Comment by dustymabe** [comment]
We discussed this in the [community meeting yesterday](https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/message/S5JTEOHJ2MWAZOWAVVO7E22SGJHO2JSI/).
We went over newly added items in the changeset (126->132 and 217). I have updated the description with the results.

**Comment by dustymabe** [comment]
Reran the script and updated the description with new changes that have come in.

**Comment by lucab** [comment]
We ran through the new items and found some actionable items:
```
* jlebon to ensure there are FCOS/Ignition ticket to track the preset changes
* lucab to check if/where we want to start using 'restorecon -T 0' in our initramfs logic
* travier to split off selinux and ostree/fcos concerns to a new ticket/PR, mainly for docs purposes
```

**Comment by dustymabe** [comment]
We discussed this in the [community meeting today](https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/message/EZUZZECUFCK7AHY54DORSXGC3KZHG6MW/).

```
12:38:22   dustymabe | ok a few updates on this topic.. the macaddresspolicy
                     | change fell out (thomas got busy and went on vacation)
                     | but we'll get it into F38. https://fedoraproject.org/w
                     | iki/Changes/MAC_Address_Policy_none


12:39:11   dustymabe | #info the hostname change got into f38/f37: https://gi
                     | thub.com/coreos/fedora-coreos-tracker/issues/902#issue
                     | comment-1225839825

```

We also covered the new self contained changes since the last time.
All of them have no impact because we don't ship them.

**Comment by dustymabe** [comment]
Reran the script and updated the description with new changes that have come in.

**Comment by cgwalters** [comment]
I believe this is done.
